libraries/AP_InertialSensor/AP_InertialSensor_BMI088.cpp

```diff
@@ -200,15 +200,28 @@ bool AP_InertialSensor_BMI088::accel_init()
     // dummy ready on accel ChipID to init accel (see section 3 of datasheet)
     read_accel_registers(REGA_CHIPID, &v, 1);
 
-    if (!read_accel_registers(REGA_CHIPID, &v, 1) || v != 0x1E) {
+  if(read_accel_registers(REGA_CHIPID,&v,1)){
```

Comment:
Indenting needs fixing.  Please revert this to just early-return-false in the case the `read_accel_register` fails, then put your switch here after it.

Also, we're pretty strict on the formatting, so:
```suggestion
  if (!read_accel_registers(REGA_CHIPID, &v, 1)) {
```

Reply:
Thank you @peterbarker, I'll commit these changes.